### PR TITLE
Updated changelog for 0.4.0 release

### PR DESCRIPTION
**Pull request by giubacc**
Updated changelog for 0.4.0 release

Signed-off-by: Giuseppe Baccini <giuseppe.baccini@suse.com>

### CODE REVIEW

**Review by jhmarina — APPROVED** [review]
lgtm!

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,13 +5,23 @@ All notable changes to this project will be documented in this file.
...
+- added `tests/s3gw-buckets-rest-api-test.py` to test bucket related rest calls.
+
+### Changed
+
+- relocate docs to [s3gw repository](https://github.com/aquarist-labs/s3gw/docs).
   import local s3gw and s3gw-ui images into a running K3s.
 
+### Fixed
+
+- Boost system packages updated to version 80_0 in gateway's Dockerfiles
```
Comment (jecluis) [inline CHANGELOG.md:23]: shouldn't this be in the 'Changed' section though?

Comment (giubacc) [reply CHANGELOG.md:23]: It depends how you want to see it; images were not building, updating packages to version 80_0 fixed the issue.

Comment (jecluis) [reply CHANGELOG.md:23]: Right, that's a fair point. But it's still an update of our dependencies, not a bug.

**Comment by giubacc** [comment]
merging because we are near to release

**Pull request merged**